src/deadline/houdini_submitter/otls/deadline_cloud.hda/Driver_1deadline__cloud/DialogScript

```diff
@@ -736,6 +751,7 @@
                 type    directory
                 default { "" }
                 parmtag { "script_callback_language" "python" }
+                help    "Directories where output files will downloaded."
```

Comment:
I think these are the directories whose outputs are uploaded right? Like if the scene renders to a file and the file is not in one of these directories, it doesn't get uploaded?

Reply:
It's both-ish. `The location where outputs would be created if rendered locally` is probably more accurate. 

If you choose to use job attachments, then current implementation-wise these locations are then:
1.  tracked on the worker (after path mapping) such that any file created there after a certain time will be uploaded to S3 as output.
2. where outputs in s3 are downloaded to locally when running the download command.

Reply:
That's true! Updated to `Directories that will contain output files when the job is run. This controls what the worker will upload after rendering.`